### DISCUSSION TITLE
it would be great if clickhouse-Iceberg(S3) could support partition filtering

### DISCUSSION THREAD

**Discussion by freeman-buzzvil** [ClickHouse/ClickHouse#73258] [Ideas]
Compared to plain Parquet, Iceberg offers flexible schema alterations and comes with built-in dynamic partitioning, which has been extremely useful for most of production operation.

Recently, I tried to read an Iceberg table stored on S3 from ClickHouse. While the table creation worked without any issues.
(but later I realized current CH-iceberg cannot read schema altered table)

I encountered a serious problem during SELECT queries, as the lack of partition filtering caused the query to attempt to load all the data—exceeding hundreds of gigabytes

**Comment by melvynator** [comment]
We added support for it: https://github.com/ClickHouse/ClickHouse/pull/72044

**Reply by freeman-buzzvil** [reply]
Thank you, I'll check it out.
